### PR TITLE
Fix timestamp and datetime fields in \ccxt\async\Exchange.php

### PR DESCRIPTION
**Pull request by escoman**

### CODE REVIEW

**Review by pcriadoperez — COMMENTED** [review]
Great catch @escoman!

This part of the file is transpiled from `Exchange.js`, could you make the change there so it applies to all languages?

**Comment by escoman** [comment]
@pcriadoperez, sorry.
I only use the php-version of the library.

**Comment by escoman** [comment]
@pcriadoperez, I tried. I hope it's successful.

File: js/base/Exchange.js [hunk 1]
```diff
@@ -1206,8 +1206,8 @@ module.exports = class Exchange {
         return this.extend (order, {
             'id': this.safeString (order, 'id'),
             'clientOrderId': this.safeString (order, 'clientOrderId'),
-            'timestamp': datetime,
-            'datetime': timestamp,
+            'timestamp': timestamp,
```
Comment (sc0Vu) [inline js/base/Exchange.js:1209]: Hi @escoman

The `safeOrder` function will be compiled from js to python/php. Would you like to keep js updates?

BTW, you can update local php library with `npm run build` (if node was installed).

Comment (escoman) [reply js/base/Exchange.js:1209]: @sc0Vu, hello.

So, for the correct changes, I have to correct [js/base/exchange.js], and then start `npm run build`, which will make changes to Python/PHP?

Comment (carlosmiei) [reply js/base/Exchange.js:1209]: @escoman all good now, will be merged soon thanks!

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**